### PR TITLE
Reorganise /paths into individual files

### PR DESCRIPTION
**Pull request by ben-ahmady-cko**
I'm proposing what I *hope* is a useful reorganisation of the `/paths` data in our OpenAPI spec. It's outside the scope of just tech writers so do please tell me who needs to review. If we decide not to do this that's totally fine, but I'd like to discuss it :D 

Pros:
* We get more control over the order in which endpoints appear in the reference and in the Postman collection we'll be using this spec to autogenerate—currently it's all automatically alphabetical and I'm not aware of a way within OpenAPI to enforce manual organisation
* Slightly easier file organisation 

Cons I am aware of:
* People might be used to the current system, so some minor mistakes may arise

### CODE REVIEW

**Comment by ben-ahmady-cko** [comment]
The reason this is failing is because I haven't modified the NAS spec yet: I didn't want to spend time doing that before we agreed this was a good approach, and I wanted to use the ABC spec as a POC

You can verify it builds successfully locally (famous last words...)

**Comment by chrisi-webster-cko** [comment]
So, essentially, instead of having multiple methods in one file, we're splitting them out? I like it.

**Comment by ben-ahmady-cko** [comment]
> So, essentially, instead of having multiple methods in one file, we're splitting them out? I like it.

That's basically it, yeah. The idea being that it gives us control over the ordering of the endpoints in the things we'll use the spec for (API reference, Postman, and maybe more).

**Review by chrisi-webster-cko — APPROVED** [review]

**Review by cristina-szilagyi-cko — APPROVED** [review]

File: abc_spec/paths/customers.yaml [hunk 1]
```diff
@@ -1,29 +1,136 @@
...
+            schema:
+              $ref: '#/components/headers/Cko-Request-Id'
+          Cko-Version:
+            schema:
+              $ref: '#/components/headers/Cko-Version'
+      '401':
+        description: Unauthorized
+      '404':
+        description: Customer not found
+/customers/{id}:
```
Comment (jack-hutchinson-cko) [inline abc_spec/paths/customers.yaml:77]: Why do you have `/customers` and `/customers/{id}` in one file. But for instance, disputes are separated into individual files? 
`/disputes/`
`/disputes/{dispute_id}`

**Review by jack-hutchinson-cko — APPROVED** [review]
LGTM 👍

**Review by chrisi-webster-cko — APPROVED** [review]
Reviewed changes to nas_spec files

**Pull request merged**